### PR TITLE
Opinionated refactor of the Logo

### PR DESCRIPTION
**Pull request by unseen-ninja**
**Description**
This PR changes the M-Logo assets to a redrawn version which edges out a couple of the quirks I personally found to be offputting; The misaligned strokes and different stroke widths of the letter to be precise. I personally feel like this helps to make the logo look more cleaned up in general and achieves a calmer look because it's not all over the place with it's strokes and stroke-placements.

**Screenshots**
The old version on the left, the new one on the right.
image

### CODE REVIEW

File: assets/meowix.ico
Comment (Nekosis) [inline assets/meowix.ico:1]: Please add an ICO version of the new icon. [This is used for the website.](https://github.com/Meowix-Linux/meowix-linux.github.io/blob/20cdf590210daa7abaf18d59539cfc3352c81638/index.html#L15C1-L19C7)

**Review by Nekosis — DISMISSED** [review]
Overall, this new logo looks great, but we do need an ICO version of the logo as it is used for the website. If you know a way we can instead use the PNG or SVG, let me know. Not very experienced with HTML.

Thank you so much for your awesome contribution!

**Comment by Nekosis** [comment]
So sorry for the confusion with the ICO. This pull request will be merged now!

Thank you so much @andreasgrafen for contributing!

**Comment by andreasgrafen** [comment]
Absolutely no worries! Confusion happens …

If you don't necessarily care about legacy support for stoneage browsers you can simply use a PNG as the favicon on the website by just using:
```html
<link rel="shortcut icon" type="img/png" href="path/to/favicon.png">
```

**Pull request merged**